Automattic/sensei
# Add Quick Links section

Fixes #5626

### Changes proposed in this Pull Request

* Add Quick Links section components;

### Testing instructions

On a WP installation containing this branch of the Sensei code:

1. Click on Sensei LMS -> Home;
2. Verify if the Quick Links section appear in the same way as in the image below;
3. Verify if, when you click on "install demo course", the label of the link changes accordingly;
4. Verify if the links work correctly;

### Screenshots/Videos

![Image of the Quick Links section](https://user-images.githubusercontent.com/529864/194966283-6d3e89ea-ffc7-4121-8a21-2ba6ef81f95b.png)
